packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/HelloConstructProps.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelloCdk
+{
+    public class HelloConstructProps
```

Comment:
I would recommend placing this class alongside `HelloConstruct`. As far as I know .NET doesn't require a class-per-file and moreover, encourages co-locating related classes, especially like this one.

Reply:
It doesn't require class per file. However, co-location is a judgement call. In this case, I can agree that your suggestion makes sense. In practice, I usually do one class per file anyway.